cmd/process-agent/command/main_common.go

```diff
@@ -184,7 +185,8 @@ func runApp(ctx context.Context, globalParams *GlobalParams) error {
 		}),
 
 		// Provides specific features to our own fx wrapper (logging, lifecycle, shutdowner)
```

Comment:
Is this comment still true (i.e. should we remove `logging`) ?

Reply:
This comment is still true. There are two ways to log fx events either with `TRACE_FX:1` or `log_level:debug`